examples/gpt-2/README.md

```diff
@@ -122,7 +139,56 @@ in this way".
 
 ## Quick Start (II) - Fine-tune the Pre-trained Model 
 
-Coming soon!
+This section shows how we can fine-tune the pre-trained GPT2 model and use the resulting model for generation.
+
+### Prepare data
+
+We first preprocess data with the GPT-2 BPE encoding. 
+
+A toy dataset is provided under [`data/toy/`](data/toy) which includes `train.txt`, `dev.txt`, and `test.txt`. This example will fit the GPT-2 model on `train.txt`, evaluate perplexity on `dev.txt`, and do continuation generation using `test.txt` as the context.
+
+Run the following cmd to transform the data into [pickle](https://docs.python.org/3/library/pickle.html) format and perform processing such as truncation, BPE encoding, adding special tokens, etc:
+
+```bash
+python prepare_data.py --data_dir data/toy \
+--max_seq_length=128 \
+--tfrecord_output_dir=data/toy \ 
```

Comment:
`tfrecord_output_dir` -- it shouldn't be `tfrecord`.

Can we just name it `output_dir`? The TF  example could be updated in the same way to make the interfaces the same